webrtc.html

```diff
@@ -2849,6 +2849,16 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       Each track to be sent is associated with exactly one <code><a>RTCRtpSender</a></code>, and
       each track to be received is associated with exactly one <code><a>RTCRtpReceiver</a></code>.</p>
 
+      <p>The basic principle is that the encoding and transmission of each
+      <code>MediaStreamTrack</code> is made such that the characteristics are as far as possible
+      maintained by the track created on the remote side. In other words, the
+      <code>MediaTrackSettings</code> width, height, frameRate for a video track, and volume, sampleSize,
+      sampleRate and channelCount for an audio track should be maintained. There are situations when this
+      does not apply, there may for example be resource constraints (HW Codec, CPU, network throughput
+      etc.) that makes this impossible, or there may be <code><a>RTCRtpSender</a></code> settings applied
```

Comment:
"Impossible" sounds like a high bar. I could see UAs faced with lots of complicated decisions about how to best allocate bandwidth between competing demands. If we removed "that makes this impossible", then the sentence still works I think fwiw.